### PR TITLE
Initial AppVeyor support

### PR DESCRIPTION
**Pull request by chipitsine**

### CODE REVIEW

**Comment by mattock** [comment]
FYI: I enabled AppVeyor for tap-windows6.

Have you tested that .appveyor.yml works in your own fork?

**Comment by chipitsine** [comment]
https://ci.appveyor.com/project/chipitsine/tap-windows6/build/1.0.28


I created that file using appveyor --> "download yaml"

Ok, I'll play with .gitattributes to fix that

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.yml text=auto
```
Comment (mattock) [inline .gitattributes]: This is probably a good idea for .yml files :+1:

**Comment by mattock** [comment]
Can you squash the commits together and force-push? Then this is ready to merge.

**Comment by chipitsine** [comment]
squashed

**Pull request closed without merging**